src/transport/IonGasTransport.cpp

```diff
@@ -45,6 +45,19 @@ void IonGasTransport::init(thermo_t* thermo, int mode, int log_level)
             m_kNeutral.push_back(k);
         }
     }
+    // set up O2/O2- collision integral [A^2]
+    // Data taken from:
+    // Prager, Jens. Modeling and simulation of charged species in
+    // lean methane-oxygen flames. Diss. 2005. Page 104.
```

Comment:
I think the better place to document this (in full) would be in the docstring for the `IonTransport` class, so that this information appears in the documentation, then here you can just write that these are from "Prager (2005)".